eng/pipelines/templates/steps/analyze.yml

```diff
@@ -87,6 +87,24 @@ steps:
      scriptPath: 'scripts/devops_tasks/setup_execute_tests.py'
      arguments: '"$(TargetingString)" --service=${{parameters.ServiceDirectory}} --toxenv=verifywhl'
 
+  - task: PowerShell@2
+    displayName: 'Run Nightly SmokeTests'
+    inputs:
+      targetType: 'filePath'
+      filePath: scripts/devops_tasks/Smoke-Tests-Nightly.ps1
+      pwsh: true
+      arguments: '${{ parameters.ServiceDirectory }}'
+      condition: and(eq(variables['Build.Reason'],'Schedule'))
+
+  - task: PowerShell@2
+    displayName: 'Run Release SmokeTests'
```

Comment:
I would recommend putting those into jobs/ci.yml 's build stage, right before regression test. And instead of a step, I think running it as a separate job will be better/more visible on the pipeline.
https://github.com/Azure/azure-sdk-for-python/blob/main/eng/pipelines/templates/jobs/ci.yml#L178